guides/common/modules/snip_warning-destroy-vm-on-host-delete.adoc

```diff
@@ -0,0 +1,8 @@
+[WARNING]
+====
+By default, if a host entry that is associated with a virtual machine is deleted, the virtual machine will remain on the compute resource.
```

Comment:
```suggestion
By default, deleting a host entry associated with a virtual machine does not delete the virtual machine on the compute resource.
```
Suggestion for removing passive voice and future tense.

Reply:
A small suggestion:
```suggestion
By default, deleting a host entry associated with a virtual machine does not destroy the virtual machine on the compute resource.
```